internal/nostr/nostr.go

```diff
@@ -650,6 +647,12 @@ func (svc *Service) startSubscription(ctx context.Context, subscription *Subscri
 				time.Sleep(5 * time.Second) // sleep for 5 seconds
 				break
 			} else {
+				if (subscription.RequestEvent != nil) {
```

Comment:
I saw above there is a nested for loop in startSubscription, is it really needed?

Reply:
Whoops yes, it's not needed, we just have to `continue` so that we loop again to get the relay connection and try again